### PR TITLE
CloudService 2022-09-04 

### PR DESCRIPTION
**Pull request by grizzlytheodore**
SDK PR for CloudServiceRP 2022-09-04 https://github.com/Azure/azure-rest-api-specs/pull/21630

# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by grizzlytheodore** [comment]
adding 'do not merge' tag until Swagger PR is merged https://github.com/Azure/azure-rest-api-specs/pull/21630

**Comment by grizzlytheodore** [comment]
@ArcturusZhang, how do I resolve failing Mac/Linux CI checks here? 

##[error]sdk/extendedlocation/Microsoft.Azure.Management.ExtendedLocation/tests/Microsoft.Azure.Management.ExtendedLocation.Tests.csproj(0,0): Error NU1504: Duplicate 'PackageReference' items found. Remove the duplicate items or use the Update functionality to ensure a consistent restore behavior. The duplicate 'PackageReference' items are: xunit 2.3.1, xunit 2.4.1.
22:53:04.623   204>Project "/mnt/vss/_work/1/s/sdk/extendedlocation/Microsoft.Azure.Management.ExtendedLocation/tests/Microsoft.Azure.Management.ExtendedLocation.Tests.csproj" (204) is building "/mnt/vss/_work/1/s/sdk/extendedlocation/Microsoft.Azure.Management.ExtendedLocation/tests/Microsoft.Azure.Management.ExtendedLocation.Tests.csproj" (204:3) on node 1 (_GenerateRestoreProjectPathWalk target(s)).
22:53:04.634   204>/mnt/vss/_work/1/s/sdk/extendedlocation/Microsoft.Azure.Management.ExtendedLocation/tests/Microsoft.Azure.Management.ExtendedLocation.Tests.csproj : error NU1504: Duplicate 'PackageReference' items found. Remove the duplicate items or use the Update functionality to ensure a consistent restore behavior. The duplicate 'PackageReference' items are: xunit 2.3.1, xunit 2.4.1.
22:53:04.634   204>Done Building Project "/mnt/vss/_work/1/s/sdk/extendedlocation/Microsoft.Azure.Management.ExtendedLocation/tests/Microsoft.Azure.Management.ExtendedLocation.Tests.csproj" (_GenerateRestoreProjectPathWalk target(s)) -- FAILED.
22:53:04.635   204>Done Building Project "/mnt/vss/_work/1/s/sdk/extendedlocation/Microsoft.Azure.Management.ExtendedLocation/tests/Microsoft.Azure.Management.ExtendedLocation.Tests.csproj" (Restore target(s)) -- FAILED.

**Comment by grizzlytheodore** [comment]
Adding 'breakingChangedApproved' and 'ARMSigned' because 
Only one PR(https://github.com/Azure/azure-rest-api-specs/pull/21232) added for this release and the reviews were made in that PR. 

 Breaking change issues justification: 
RemovedProperty and AddedRequiredProperty come from updates in the capitalization in the properties:
frontendIpConfigurations -> frontendIPConfigurations

It was updated because the API was returning the second, and the swagger was defined as the first and some SDKs were having issues.

ChangedParameterOrder is from addressing modelValidation issues. It was complaining that parameters have to be in the order of the path defined so they updated the order to meet it.

**Comment by grizzlytheodore** [comment]
@ArcturusZhang

File: sdk/compute/Microsoft.Azure.Management.Compute/src/Microsoft.Azure.Management.Compute.csproj
Comment (ArcturusZhang) [inline sdk/compute/Microsoft.Azure.Management.Compute/src/Microsoft.Azure.Management.Compute.csproj:31]: Should we make some updates in this file on the version number and release notes?

Comment (grizzlytheodore) [reply sdk/compute/Microsoft.Azure.Management.Compute/src/Microsoft.Azure.Management.Compute.csproj:31]: we should. you're right. let me do that

Comment (grizzlytheodore) [reply sdk/compute/Microsoft.Azure.Management.Compute/src/Microsoft.Azure.Management.Compute.csproj:27]: all updated

**Comment by grizzlytheodore** [comment]
@ArcturusZhang  please review/merge

**Review by ArcturusZhang — APPROVED** [review]

**Pull request merged**
